File: Array/unique.cpp
#include<iostream>
using namespace std;
void unique(int arr[],int n)
{
    for(int i =0;i<n;i++)
    {
        for(int j = i+1;j<n;j++)
        {
            if(arr[i] == arr[j])
            {
                
            }
        }
    }
}
int main()
{
    int n;
    cin >> n;
    int arr[n];
    for(int i =0;i<n;i++)
    {
        cin >> arr[i];
    }
    unique(arr,n);
}

File: Array/unique.py
mylist = [0,1,3]

def unique(mylist):
    
    a = 0
    for i in range(len(mylist)):
        for j in range(len(mylist)):
            if mylist[i] != mylist[j]:
                a = mylist[i]
    return a
print(unique(mylist))

    
    
    
    
    
    
    